website/.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "workbench.colorCustomizations": {
+    "statusBar.foreground": "#000",
+    "statusBar.background": "#19C6CB"
+  },
+}
```

Comment:
Given that these are the same as the top-level ones, maybe we stick with just those for now?

Reply:
Good point but I believe some people might work out of just the `/website` directory so it was suggested to add it in here too

Reply:
I will confirm this and report back

Reply:
Ah OK, that makes sense, that hadn’t occurred to me.

Reply:
Given that rationale, I’m going to resolve this.